### ISSUE TITLE
Planning on adding the upgraded features farshadz1997 added?

### ISSUE THREAD

**Issue by Vxlpix** [charlesbel/Microsoft-Rewards-Farmer#228]
discord, shuffle etc

**Comment by charlesbel** [comment]
You're welcome to make a PR if you want, might be done if I find the time to do it

**Comment by udu3324** [comment]
I can go ahead and do the discord one

**Comment by ShoGinn** [comment]
Do you know if anyone still has the farshadz1997 code?

**Comment by udu3324** [comment]
> Do you know if anyone still has the farshadz1997 code?

Yeah its still [here](https://gitlab.com/farshadzargary1997/Microsoft-Rewards-bot)

**Comment by charlesbel** [comment]
It's now done !
